moderators.md

```diff
@@ -0,0 +1,137 @@
+Each incident, the actions taken to respond to the incident, and the resolution must be reported as an issue at <https://github.com/r-multiverse/help/issues> with the label "policy violation".
+If you do not feel comfortable taking direct action yourself, contact an [administrator](team.md#administrators) immediately and ask the [administrator](team.md#administrators) handle the situation.
+
+1. In case of an extreme policy violation, or repeated violations of the [code of conduct](conduct.md),
+  block the user from [R-multiverse GitHub repositories](https://github.com/r-multiverse)
+  by adding the GitHub username to the [list of blocked users](https://github.com/organizations/r-multiverse/settings/blocked_users).
+1. If an R-multiverse package or [R-multiverse stakeholder](governance.md#r-multiverse-stakeholders) violates any applicable laws, contact the appropriate law enforcement agency.
+1. If an R-multiverse package on GitHub and violates any [GitHub policies](https://docs.github.com/en/site-policy),
+  [report the violation to GitHub](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
+1. If an R-multiverse package on GitLab and violates any [GitLab policies](https://handbook.gitlab.com/handbook/legal/#gitlab-policies)
+  such as [Acceptable Use](https://handbook.gitlab.com/handbook/legal/acceptable-use-policy/),
+  [report the violation to GitLab](https://docs.gitlab.com/ee/user/report_abuse.html).
+1. If an R-multiverse package has a security vulnerability such as malware,
+  report all relevant versions of the package to the [R Advisory Database](https://github.com/rconsortium/r-advisory-database).
+
+If a package violates any R-multiverse [policy](policies.md) (for example, if the [license](https://en.wikipedia.org/wiki/Free_and_open-source_software) is invalid ^[ [This list](https://github.com/r-multiverse/community/blob/main/nonstandard_licenses.json) automatically track packages without valid [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) licenses.
+[Moderators](governance.md#moderator) should block each package in this list until the package authors add a valid [FOSS](https://en.wikipedia.org/wiki/Free_and_open-source_software) license and resolve all other [policy](policies.md) violations.
+]), a [moderator](governance.md#moderator) must block it from R-multiverse:
+
+1. Locate the [package listing](https://github.com/r-multiverse/contributions/tree/main/packages)
+  in the [contributions repository](https://github.com/r-multiverse/contributions).
+2. Replace the URL or custom JSON in the listing with free-form text (not a URL or parseable JSON) explaining why the package was blocked.
+  ^[If the listing cannot be parsed as a URL or JSON, then a bot will automatically remove the package from R-multiverse infrastructure.]
+3. Contact the package authors (confidentially if needed) to inform them of these actions.
+
```

Comment:
You should include an offboarding process.  What happens when a moderator wants to stop being a moderator?

Reply:
110290207eff403c5acafcf863e4b97b26087d8d adds an offboarding process and clarifies the onboarding process.